Doc/library/os.rst

```diff
@@ -4280,6 +4275,20 @@ to be ignored.
    :func:`sys.stdout.flush` or :func:`os.fsync` before calling an
    :func:`exec\* <execl>` function.
 
+   *Arguments*
+
+   The various :func:`exec\* <execl>` functions take a list of arguments for the new
+   program loaded into the process.  In each case, the first of these arguments must
+   be the name of the program itself, and not an argument that a user may
+   have typed on a command line.
+
+   For the C programmer, this is the ``argv[0]``
```

Comment:
I decided to retain the paragraph break and instead reworded the first sentence in the paragraph.  I'm not entirely sure what the documentation is getting at with the second sentence in the paragraph.  I'm open to further suggestions for improving this paragraph - I feel it could be improved but I'm not sure how.